llumnix/entrypoints/vllm/api_server.py

```diff
@@ -241,22 +242,31 @@ async def is_ready():
         # Launch the ray cluster for multi-node serving.
         launch_ray_cluster(args.ray_cluster_port)
 
+    # Connect to a ray cluster.
+    head_node_ip = os.getenv('HEAD_NODE_IP')
+    connect_to_ray_cluster(head_node_ip, args.ray_cluster_port, namespace="llumnix")
```

Comment:
I think connect_to_ray_cluster(args.ray_cluster_port) is better, consistent with launch_ray_cluster(args.ray_cluster_port) .

Reply:
get it